src/functions.c

```diff
@@ -372,6 +375,147 @@ void fcallroCommand(client *c) {
     fcallCommandGeneric(c, 1);
 }
 
+/*
+ * FUNCTION DUMP
+ *
+ * Returns a binary blob representing all the functions.
+ * Can be loaded using FUNCTION RESTORE
+ */
```

Comment:
Consider describing how we construct the binary blob and why it requires an explicit CRC64 appended to it.

Reply:
Thanks @yossigo, updated the documentation.